source/apiVolontaria/volunteer/tests/tests_view_Participations.py

```diff
@@ -178,6 +177,99 @@ def test_create_new_participation(self):
             'attributes : {0}'.format(attributes),
         )
 
+    def test_create_new_participation_with_user_no_perms(self):
+        """
+        Ensure we cannot create a new participation
+        with a cusotm user if no admin
+        """
+        subscription_date = timezone.now()
+
+        data = {
+            'event': self.event.id,
+            'user': self.user2.id,
+            'standby': False,
+            'subscription_date': subscription_date,
+        }
+
+        self.client.force_authenticate(user=self.user)
+
+        with mock.patch('django.utils.timezone.now') as mock_now:
+            mock_now.return_value = subscription_date
+            response = self.client.post(
+                reverse('volunteer:participations'),
+                data,
+                format='json',
+            )
+
+        content = json.loads(response.content)
+
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertEquals(content['detail'],
```

Comment:
`detail` keyword is not accurate here since it's directly linked to the `user` field : `content['detail']`